From 06980c58667670515d9139c97809cef0a7c2c72d Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 17 Oct 2022 16:10:53 +0200
Subject: [PATCH] ARM: dts: stm32: add exti entry 43 for USBH_EHCI on
 stm32mp151

Add exti entry 43 used for USBH_EHCI wake up.
This event is mapped to GIC 75.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I1e81dabba6da65bc55725c08b79c35f4e90a39b3
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/st/stm32mp151.dtsi b/arch/arm/boot/dts/st/stm32mp151.dtsi
index f4c218e3674f..21bc8a13aca2 100644
--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -1264,6 +1264,7 @@ exti-interrupt-map {
 					<31 0 &intc GIC_SPI 53  IRQ_TYPE_LEVEL_HIGH>,
 					<32 0 &intc GIC_SPI 82  IRQ_TYPE_LEVEL_HIGH>,
 					<33 0 &intc GIC_SPI 83  IRQ_TYPE_LEVEL_HIGH>,
+					<43 0 &intc GIC_SPI 75  IRQ_TYPE_LEVEL_HIGH>,
 					<47 0 &intc GIC_SPI 93  IRQ_TYPE_LEVEL_HIGH>,
 					<48 0 &intc GIC_SPI 138 IRQ_TYPE_LEVEL_HIGH>,
 					<50 0 &intc GIC_SPI 139 IRQ_TYPE_LEVEL_HIGH>,
-- 
2.39.5

